### ISSUE TITLE
How "Split the training set into training and validation set according to the protocol here"?

### ISSUE THREAD

**Issue by lihua213** [qianguih/voxelnet#40]
Is there a detailed script for splitting them? Thanks
